src/scene/text/canvas/CanvasTextSystem.ts

```diff
@@ -138,249 +104,6 @@ export class CanvasTextSystem implements System
         TexturePool.returnTexture(texture);
     }
 
-    /**
-     * Renders text to its canvas, and updates its texture.
-     *
-     * By default this is used internally to ensure the texture is correct before rendering,
-     * but it can be used called externally, for example from this class to 'pre-generate' the texture from a piece of text,
-     * and then shared across multiple Sprites.
-     * @param text
-     * @param style
-     * @param resolution
-     * @param canvasAndContext
-     */
-    public renderTextToCanvas(text: string, style: TextStyle, resolution: number, canvasAndContext: CanvasAndContext): void
```

Comment:
If this is a public API we should keep it around and call `CanvasTextGenerator` under the hood for compatibility